Source/WebKit/UIProcess/Inspector/WebInspectorUIProxy.cpp

```diff
@@ -594,7 +610,7 @@ void WebInspectorUIProxy::closeFrontendPageAndWindow()
 
 void WebInspectorUIProxy::sendMessageToBackend(const String& message)
 {
-    auto inspectedPage = this->inspectedPage();
+    RefPtr inspectedPage = m_inspecedPage.geet();
```

Comment:
typo

Reply:
Oh weird. I swear I had built this but I guess not.